QuickStart/ChatViewController.swift

```diff
@@ -6,17 +6,41 @@
 //
 
 import SendBirdDesk
+import SendbirdChatSDK
 import SendbirdUIKit
 import UIKit
 
 class ChatViewController: SBUGroupChannelViewController {
     var ticket: SBDSKTicket!
     
+    /// When this flag is set to true, system messages will be hidden from the message list view.
+    static let shouldHideSystemMessages = false
```

Comment:
@mininny Doesn't it need to hide as a default?

Reply:
I think it should be false, because the default behavior should be to show system messages. If a user wants to hide the system meessage, they can refer to this variable and set it to true